conans/client/profile_loader.py

```diff
@@ -117,6 +120,11 @@ def read_profile(profile_name, cwd, default_folder):
     logger.debug("PROFILE LOAD: %s" % profile_path)
     text = load(profile_path)
 
+    if profile_name.endswith(".tpl"):
```

Comment:
Why that extension? Might be `.jinja`?
https://jinja.palletsprojects.com/en/3.0.x/templates/#template-file-extension

Reply:
Yes, probably better to use ``jinja``.
But this was only an implementation trick, maybe it is not necessary to use the extension, other alternatives:

- Render always as a template. Jinja syntax is not valid at the moment, so sounds it won't be breaking.
- Use other argument like ``--profile-template=myprofile``
- Use a header first line in profile files like ``# jinja2 template`` (this could be good to later model other templating mechanisms or parameters of the rendering)

Reply:
> Render always as a template. Jinja syntax is not valid at the moment, so sounds it won't be breaking.

It shouldn't but...might it be some profile with `{%` or `{{` ?

Probably the simpler to implement and maintain and more user-friendly was the extension.